src/pages/layer-2.js

```diff
@@ -679,6 +679,15 @@ const Layer2Page = ({ data }) => {
         <p>
           <Translation id="layer-2-use-layer-2-1" />
         </p>
+        <p>
+          NOTE: When bridging over and using layer 2, it is important to note
+          that you will control the address for you EOA account just like on
+          Ethereum Mainnet. However, if you are using a smart contract wallet,
+          you will not have control over this address on a layer 2 by default
+          unless deployed using `CREATE2`. If you are bridging or sending funds
```

Comment:
> unless deployed using `CREATE2`

wouldn't you need to also redeploy on the L2 before? If there is an extra step to take we should maybe list it or simplify the sentence.